test/jdk/java/awt/Choice/DisabledList.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Checkbox;
+import java.awt.Choice;
+import java.awt.Frame;
+import java.awt.Window;
+import java.awt.event.ItemEvent;
+
+/*
+ * @test
+ * @bug 6476183
+ * @summary Drop down of a Choice changed to enabled state has a disabled like appearance
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual DisabledList
+ */
+
+public class DisabledList {
+
+    private static final String INSTRUCTIONS = """
+            1) Switch checkbox
```

Comment:
Shouldn't it be "Enable checkbox"?

Reply:
I would read the "Enable checkbox" as somehow related to `setEnabled`, but the component is already enabled.
So I'll keep it as it is.

Reply:
As per Checkbox spec
"A check box is a graphical component that can be in either an "on" (true) or "off" (false) state"
so I guess "Switch on checkbox" would be more apt.. To me atleast, "switch checkbox" doesn't sound perfect...

Reply:
It is off by default in this test, so "switch" will set it to the desired selected state. So it should be OK.
Anyway, updated to the "Select the checkbox"